docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,657 @@
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "entities = ['O', 'MISC', 'PER', 'ORG', 'LOC'] \n",
+    "\n",
+    "def color_sentence(sentence, word): \n",
+    "    start_idx = sentence.index(word) \n",
+    "    before, after = sentence[:start_idx], sentence[start_idx + len(word):]\n",
+    "    return '%s%s%s' % (before, colored(word, 'red'), after) \n",
+    "\n",
+    "def print_issue(issue, show_labels=True): \n",
+    "    i, j = index_to_tuple[issue] \n",
+    "    issue_label = entities[labels[issue]] \n",
+    "    predicted_label = entities[np.argmax(pred_probs[issue])] \n",
+    "    issue_word = given_words[i][j] \n",
+    "    print('%s' % color_sentence(sentences[i], issue_word)) \n",
+    "    if show_labels: \n",
+    "        print('Given label: %s, suggested label: %s\\n' % (issue_label, predicted_label)) \n",
+    "\n",
+    "for idx, issue in enumerate(issues[:top]): \n",
+    "    print('%d.' % (idx+1), end=' ') \n",
+    "    print_issue(issue) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "96d04902",
+   "metadata": {},
+   "source": [
+    "Let’s zoom into some specific examples from the above: \n",
+    "\n",
+    "Given label is `ORG` but should be `O`: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "ddf4edf9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "A Reuter consensus survey sees medical equipment group Radiometer reporting largely unchanged earnings when it publishes first half 19996/97 results next \u001b[31mWednesday\u001b[0m.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print_issue(issues[0], show_labels=False) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3068a45b",
+   "metadata": {},
+   "source": [
+    "Given label is `MISC` but should be `LOC`: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "id": "61b6ecca",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "-- \u001b[31mBangkok\u001b[0m newsroom (662) 652-0642\n"
+     ]
+    }
+   ],
+   "source": [
+    "print_issue(issues[6], show_labels=False) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07a328ca",
+   "metadata": {},
+   "source": [
+    "Given label is `MISC` but should be `LOC`: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "557fafe6",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The basket comprises Algeria's Saharan Blend, Indonesia's Minas, Nigeria's Bonny Light, Saudi Arabia's Arabian Light, \u001b[31mDubai\u001b[0m of the Uae, Venezuela's Tia Juana and Mexico's Isthmus.\n"
+     ]
+    }
+   ],
+   "source": [
+    "print_issue(issues[8], show_labels=False) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d213b2b2",
+   "metadata": {},
+   "source": [
```

Comment:
I don't think we need this section, given the user can just inspect the sentences in the above section more closely comparing the cleanlab-suggested label and the given label, and deciding for themself if this is reasonable or not.

Reply:
Instead I would replace this section with a reminder to the user what each class name represents (the abbreviations are a bit hard to recall)

Reply:
I wanted to show that the issues are not guaranteed to be accurate or can be ambiguous, so may require manual checking (like the ones in the MNIST tutorial)

Reply:
To make it more clear, maybe explicitly state this, and just show 1 example that cleanlab correctly identified an issue in. Also try to just briefly reference the sentence above rather than restating the entire sentence (and just state the suggested vs given label discrepancy appears to be valid for this token in this sentence).